.scrutinizer.yml

```diff
@@ -1,5 +1,5 @@
 filter:
-    paths: [lib/*]
+    paths: [src/*]
```

Comment:
Changing this is a BC break for anyone not using composer.

Reply:
For 1.0 version which is `MAJOR` not ok?